drivers/hmis/lib/form_data/default/fragments/signature.json

```diff
@@ -0,0 +1,100 @@
+{
+  "type": "GROUP",
+  "variant": "SIGNATURE",
+  "link_id": "signature_group",
+  "text": "eSignature",
+  "subtitle": "The following signatures are required on this document.",
```

Comment:
This content seems a little more like helper text than a subtitle. Maybe this should be named "helper_text" instead? (We already have that property I think)